src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -137,22 +148,36 @@ public class PullRequestHandler : IHttpHandler
         return RunGitCommands(gitCommands, executionResult);
     }
 
+    private bool SyncToBranch(string branch, StringBuilder executionResult)
+    {
+        string[] gitCommands = new string[]
+        {
+            "clean -fdx",
+            "checkout master",
```

Comment:
No, we wouldn't ever use it here. This is for the merge step from origin/master, and hence this will never vary.